developers/blobstream.md

```diff
@@ -14,89 +14,111 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
-as a Celestia validator which won't be covered in the following sections
-aimed at developers.
+An implementation of Blobstream, by [Succinct](https://succinct.xyz/), called
+[Blobstream X](https://github.com/succinctlabs/blobstreamx), is out
+and will be used in our canonical deployments. This latter proves Celestia 
+block headers on the target EVM chain using zk-proofs which allows
+inheriting all the security guarantees of Celestia.
 
-## Overview
+## What is Blobstream X?
 
-Blobstream,
-consists of two components: an [orchestrator](../nodes/blobstream-orchestrator.md)
-and a [relayer](../nodes/blobstream-relayer.md).
+Blobstream X is an implementation of Blobstream with a
+ZK light client that bridges Celestia’s modular DA layer to
+Ethereum to allow high-throughput rollups to use Celestia’s DA while settling
+on Ethereum.
 
-In the following diagram, we show how a layer 2 (L2) would post data to
-Celestia and then verify that it was published in the target EVM chain.
+Optimistic or ZK rollups that settle on Ethereum but wish to use Celestia for
+DA require a mechanism for _bridging_ Celestia’s data root to Ethereum as part
+of the settlement process. This data root is used during inclusion proofs to
+prove that particular rollup transactions were included and made available in
+the Celestia network.
 
-![Blobstream-Architecture](/img/blobstream/Blobstream.png)
+Bridging Celestia’s data root to Ethereum requires running a Celestia
+_light client_ as a smart contract on Ethereum, to make the latest state
+of the Celestia chain known on Ethereum and available to rollups. Blobstream
+X utilizes the latest advances in zero-knowledge proofs to generate a
+_succinct proof_ that enough Celestia validators have come to consensus
+(according to the Tendermint consensus protocol) on a block header, and
+verifies this proof in the Blobstream X Ethereum smart contract to update
+it with the latest Celestia header.
 
-Data will first be attested to by the Celestia validator set, _i.e._
-signing commitments committing to the data. Then, these signatures will be
-relayed to the target EVM chain (in this case, Ethereum). Finally,
-the L2, or any party, will be able to verify that the data was published
-to Celestia directly on the EVM chain on the Blobstream smart contract. You can
-reference [the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
+The Blobstream X zero-knowledge proof not only verifies the consensus of
+Celestia validators, but it also merkelizes and hashes all the data roots
+in the block range from the previous update to the current update, making
+accessible all Celestia data roots (verifiable with a Merkle inclusion proof
+against the stored Merkle root) to rollups.
 
-The **orchestrator** is part of the validator setup and works as follows:
+Blobstream X is built and deployed with
+[Succinct’s unified proving stack](https://succinct.xyz/).
```

Comment:
can we change this to "Blobstream X is built and deployed with Succinct's protocol" and then link to docs.succinct.xyz? It's currently not up but I should have it up by Monday.

Reply:
[d7d2f14](https://github.com/celestiaorg/docs/pull/1379/commits/d7d2f14e872ee4fcc5a06079a3e5b0c29840e0f2)

Reply:
leaving this unresolved until docs are up, as a reminder

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/654b9134117b8fce86c94573f2340d213a55c9c3